armory/utils/config_loading.py

```diff
@@ -75,8 +76,18 @@ def load_model(model_config):
     model_module = import_module(model_config["module"])
     model_fn = getattr(model_module, model_config["name"])
     weights_file = model_config.get("weights_file", None)
+    if isinstance(weights_file, str):
+        weights_path = maybe_download_weights_from_s3(weights_file)
+    elif isinstance(weights_file, list):
+        # TODO: if there are multiple weights files,
+        #  should model_fn() be called for each
+        #  or called once on the list?
+        weights_path = [maybe_download_weights_from_s3(w) for w in weights_file]
+    else:
+        weights_path = None
```

Comment:
If weights_file is a dict, mapping from key to filenames
```
{"model1": "path1.pth", ...}
```
list logic over dict.values()

Reply:
Also ensure it is one of None, str, list, dict.